test/integration/multikueue/multikueue_test.go

```diff
@@ -187,6 +187,10 @@ var _ = ginkgo.Describe("Multikueue", ginkgo.Ordered, ginkgo.ContinueOnFailure,
 		gomega.Expect(worker2TestCluster.client.Create(worker2TestCluster.ctx, worker2Cq)).Should(gomega.Succeed())
 		worker2Lq = utiltesting.MakeLocalQueue(worker2Cq.Name, worker2Ns.Name).ClusterQueue(worker2Cq.Name).Obj()
 		gomega.Expect(worker2TestCluster.client.Create(worker2TestCluster.ctx, worker2Lq)).Should(gomega.Succeed())
+
+		ginkgo.By("Disabling AdmissionCheckValidationRules feature", func() {
+			gomega.Expect(features.SetEnable(features.AdmissionCheckValidationRules, false)).To(gomega.Succeed())
+		})
```

Comment:
I think it should be disabled by default?

Reply:
True, but I think it persists between tests, so I would rather keep it disabled before each test

Reply:
Use `features.SetFeatureGateDuringTest` it should revert after test.